ERCS/erc-7943.md

```diff
@@ -98,7 +93,7 @@ interface IERC7943 /*is IERC165*/ {
     /// @param user The address of the user.
     /// @param tokenId The ID of the token.
     /// @return amount The amount of `tokenId` tokens currently frozen for `user`.
-    function getFrozen(address user, uint256 tokenId) external view returns (uint256 amount);
+    function getFrozenTokens(address user, uint256 tokenId) external view returns (uint256 amount);
```

Comment:
At this point if these functions were `set/get Frozen` I believe it makes sense to rename `setFrozen` to `setFrozenTokens` too. Even if I believe that "tokens" is not the right word to use since ERC-721/1155 go by single units, for the sake of naming consistency I can accept that.